packages/webhooks/tasks/sendWebhook.js

```diff
@@ -151,5 +152,5 @@ async function sendWebhook (subscriptionId) {
 
 module.exports = {
     trySendData,
-    sendWebhook: createTask('sendWebHook', sendWebhook, { priority: 2 }),
+    sendWebhook: createTask('sendWebHook', sendWebhook),
```

Comment:
Same about `low` priority

Reply:
Hm...I have webhooks in multiple apps (some multi-queued, some are default only), maybe you should add wrapper similar to `getWebhookModels`, so you can specify queue as function args
<img width="490" alt="image" src="https://github.com/open-condo-software/condo/assets/50069872/4900502e-8499-4e67-9380-5d8fc5206af5">

Reply:
@SavelevMatthew I'm done that. But I would insist that you look at the solution again